### PR TITLE
adds other chrome profile directories to the collection list

### PR DESCRIPTION
**Pull request by ytonui**

### CODE REVIEW

File: osxcollector/osxcollector.py [hunk 1]
```diff
@@ -1512,12 +1520,14 @@ def ignore_db_path(sqlite_db_path):
...
-            directories_of_dbs, chrome_path, chrome_ignored_sqlite_keys,
-            ignore_db_path)
+        for profile_path in profile_paths:
+            self._log_directories_of_dbs(
+                directories_of_dbs, profile_path, chrome_ignored_sqlite_keys,
+                ignore_db_path)
 
-        with Logger.Extra('osxcollector_subsection', 'preferences'):
-            self._log_json_file(chrome_path, 'preferences')
+        for profile_path in profile_paths:
```
Comment (jjsendor) [inline osxcollector/osxcollector.py]: It looks like you have introduced 3 new for loops, each looping over the same resource. Any way you can refactor this code to merge it into 1 loop?

**Comment by ytonui** [comment]
I've updated as per review comments

File: osxcollector/osxcollector.py [hunk 2]
```diff
@@ -1491,11 +1491,7 @@ def _collect_chrome(self, homedir):
             Logger.log_warning('Directory not found {0}'.format(chrome_path))
             return
 
-        profile_paths = []
-        for subdir in os.listdir(chrome_path):
-            if os.path.isdir(os.path.join(chrome_path, subdir)):
-                if os.path.isfile("{0}/{1}/History".format(chrome_path, subdir)):
-                    profile_paths.append(pathjoin(chrome_path, subdir))
+        profile_paths = [pathjoin(chrome_path, subdir) for subdir in os.listdir(chrome_path) if os.path.isdir(os.path.join(chrome_path, subdir)) and os.path.isfile("{0}/{1}/History".format(chrome_path, subdir))]
```
Comment (jjsendor) [inline osxcollector/osxcollector.py]: Well, okay, so now the tricky question is how do you split it into multiple lines so it is readable? ;)

**Comment by jjsendor** [comment]
@ytonui for some reason Travis is failing :(
It complains about the lack of Foundation package, which is weird, because we have the Travis profile set up to build like it was an macOS environment.

**Comment by jjsendor** [comment]
👍  shipit!

**Pull request merged**